roles/valkey/vars/main.yml

```diff
@@ -33,6 +33,13 @@ _valkey_helm_values:
       repository: "{{ atmosphere_images['valkey_sentinel'] | vexxhost.kubernetes.docker_image('path') }}"
       tag: "{{ atmosphere_images['valkey_sentinel'] | vexxhost.kubernetes.docker_image('tag') }}"
     primarySet: atmosphere
+    resources:
+      requests:
+        cpu: 500m
+        memory: 512Mi
+      limits:
+        cpu: 1000m
```

Comment:
Can you test increasing the cpu limit only and nothing else?  The requests don't really matter in this case, and the memory limit is not really an issue.